From 0484f53350a00f072018b36349d13c5f2bce80c4 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@sun.com>
Date: Fri, 6 Nov 2009 16:34:51 -0800
Subject: [PATCH] Redefine "public" structure member to avoid clash with C++ keyword

Needed to prevent errors when compiling with Sun C++ compiler:
"../../include/inputstr.h", line 427: Warning: declarator required in declaration.
"../../include/inputstr.h", line 427: Error: Use ";" to terminate declarations.
"../../include/inputstr.h", line 427: Error: No ":" found following "public".
"../../include/inputstr.h", line 427: Error: "}" expected instead of ";".
"../../include/inputstr.h", line 427: Warning: declarator required in declaration.
---
 unix/xserver/hw/vnc/vncExtInit.cc |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/unix/xserver/hw/vnc/vncExtInit.cc b/unix/xserver/hw/vnc/vncExtInit.cc
index a15a77b..0364adc 100644
--- a/unix/xserver/hw/vnc/vncExtInit.cc
+++ b/unix/xserver/hw/vnc/vncExtInit.cc
@@ -24,6 +24,7 @@
 
 extern "C" {
 #define class c_class
+#define public c_public
 #define NEED_EVENTS
 #include <X11/X.h>
 #include <X11/Xproto.h>
@@ -38,6 +39,7 @@ extern "C" {
 #include "vncExt.h"
 #undef class
 #undef xalloc
+#undef public
 }
 
 #include <rfb/Configuration.h>
-- 
1.5.6.5

